aten/src/TH/generic/THTensorMath.c

```diff
@@ -10,6 +10,9 @@
 #include <omp.h>
 #endif
 
+#define HYPER_TH_OMP_OVERHEAD_THRESHOLD 2000
+#define ORDIN_TH_OMP_OVERHEAD_THRESHOLD 20000
+#define UNCERTAIN_TH_OMP_OVERHEAD_THRESHOLD 50000
```

Comment:
Why is it "uncertain"?

Reply:
You can look into this [benchmark](https://github.com/zy97140/omp-benchmark-for-pytorch), in which we just tested operations including copy, add, div, sin, exp, sum, and prod on Intel(R) Xeon(R) CPU E5-2699 v4, Intel(R) Xeon(R) Platinum 8180 CPU and Intel(R) Core(TM) i7-5960X CPU. We have not covered all operations and all CPUs yet. This uncertain threshold is generally a good one according to our experience attained from the benchmarking process.